### PR TITLE
Added support for Livewire 3

### PR DESCRIPTION
**Pull request by SebastiaanKloos**
Fixes https://github.com/getsentry/sentry-laravel/issues/787#issuecomment-1795061390

### CODE REVIEW

File: composer.json
Comment (stayallive) [inline composer.json:1]: We do this mostly for the types, so we can get autocomplete working and make phpstan a little happier without many exclusions.

File: src/Sentry/Laravel/Features/LivewirePackageIntegration.php [hunk 1]
```diff
@@ -32,11 +31,56 @@ public function isApplicable(): bool
     }
 
     public function onBoot(LivewireManager $livewireManager): void
+    {
+        if (class_exists(EventBus::class)) {
```
Comment (stayallive) [inline src/Sentry/Laravel/Features/LivewirePackageIntegration.php:35]: There is no version number, so we check for the `EventBus` class introduced in Livewire v3.

File: src/Sentry/Laravel/Features/LivewirePackageIntegration.php [hunk 2]
```diff
@@ -125,6 +202,12 @@ private function updateTransactionName(string $componentName): void
         $transaction->getMetadata()->setSource(TransactionSource::custom());
 
         Integration::setTransaction($transactionName);
+
+        $hub->configureScope(function (Scope $scope) {
+            $livewireManager = $this->container()->make(LivewireManager::class);
+
+            $scope->setTag('livewire.original_url', $livewireManager->originalUrl());
```
Comment (cleptric) [inline src/Sentry/Laravel/Features/LivewirePackageIntegration.php:209]: SDKs should not send any tags by default. Is this something a user needs to be able to search for?

Comment (SebastiaanKloos) [reply src/Sentry/Laravel/Features/LivewirePackageIntegration.php:209]: @cleptric Livewire sends state updates to a global Livewire route (`/livewire/update`) and is not very helpful to find out the exact location of an issue. The original url helps to find out on what page the issue occurred. We already improved this issue by renaming the transaction to include the Livewire component. Livewire components can be used on many pages so adding this tag makes it much easier to track down the origin of the issue.

Comment (cleptric) [reply src/Sentry/Laravel/Features/LivewirePackageIntegration.php:209]: If the transaction name is already fixed, the `transaction` value of an error should reflect this as well. So a user will see issues grouped on a route basis afaik.

Comment (stayallive) [reply src/Sentry/Laravel/Features/LivewirePackageIntegration.php]: We've decided to remove this for now and see in a follow up if we can add this parameter somewhere and make it searchable still. But we want to ship this so we'll leave it out for now.

**Review by cleptric — APPROVED** [review]

**Review by stayallive — APPROVED** [review]
💪

**Comment by cleptric** [comment]
Thanks @SebastiaanKloos and @stayallive!

**Pull request merged**